src/ui/common/src/components/integrations/dialogs/IntegrationFileUploadField.tsx

```diff
@@ -35,6 +39,9 @@ export const IntegrationFileUploadField: React.FC<
   displayFile,
   onReset,
 }) => {
+  const { control } = useFormContext();
```

Comment:
This is one approach that we can use here. Might also consider the approach used in NotificationsLevel component. It's not as clever I'd say, but I think the pattern there is simple and more in line with a KISS approach.